src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,76 @@
+package seedu.duke;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Timer;
+import java.util.TimerTask;
+
+import javafx.application.Platform;
+
+/**
+ * Main class to run Duke program.
+ * Creates and stores tasks such as todo, event and deadline.
+ */
+
+public class Duke {
+    private Storage storage;
+    private TaskList taskLists;
+    private Ui ui;
+    private Parser parser;
+
+    /**
+     * Initializes the Duke class and creates an instance of Storage, TaskList, Parser and Ui.
+     */
+    public Duke() {
+        try {
+            this.storage = new Storage();
```

Comment:
Have noticed this in a few places, the coding standard advises against the use of this if the reference is clear.